scalafmt-tests/src/test/resources/rewrite/SortModifiersInlineTransparent.stat

```diff
@@ -0,0 +1,18 @@
+maxColumn = 200 #to not disturb the output too much
+rewrite {
+  rules = [SortModifiers]
+  sortModifiers {
+    order = ["inline", "transparent"]
+  }
+}
+runner.dialect = Scala3
+<<< transparent inline method
+transparent inline def choose(b: Boolean): A = {
```

Comment:
do you need a test with all possible modifiers in the wrong order to show how it gets reordered?

Reply:
There are some tests, which don't include `transparent` or `inline`, but I think that's really not needed. Ordering is being tested anyway.